semgrep-core/tests/php/metavar_object_method.sgrep

```diff
@@ -0,0 +1 @@
+$wpdb->$METHOD(<... $ARGS ...>)
```

Comment:
this is the same content, so maybe you can use a single .sgrep, and put both test cases in the same .php file.